bindings/python/pyproject.toml

```diff
@@ -40,19 +40,34 @@ sqlalchemy = [
   "SQLAlchemy>=2.0.23",
 ]
 [tool.pdm.dev-dependencies]
-lint = ["ruff>=0.1.1"]
+lint = ["ruff>=0.1.5"]
 test = ["pytest>=7.4.3"]
 
 [tool.pdm.scripts]
-test   = "pytest tests/"
+test = "pytest tests/"
 format = "ruff format ."
-fix    = "ruff --fix ."
-check  = { composite = ["ruff format . --check", "ruff ."] }
+fix = "ruff --fix ."
+check = { composite = ["ruff format . --check", "ruff ."] }
 
 [tool.ruff]
-select = ["E", "F", "I", "TID"]
-ignore = ["E731", "E501"]
-src    = ["src"]
+select = [
+  "E", #https://docs.astral.sh/ruff/rules/#error-e
+  "F", #https://docs.astral.sh/ruff/rules/#pyflakes-f
+  "I", #https://docs.astral.sh/ruff/rules/#isort-i
+  "TID", #https://docs.astral.sh/ruff/rules/#flake8-tidy-imports-tid
+  "S", #https://docs.astral.sh/ruff/rules/#flake8-bandit-s
+  "B", #https://docs.astral.sh/ruff/rules/#flake8-bugbear-b
+  "SIM", #https://docs.astral.sh/ruff/rules/#flake8-simplify-sim
+  "N", #https://docs.astral.sh/ruff/rules/#pep8-naming-n
+  "PT", #https://docs.astral.sh/ruff/rules/#flake8-pytest-style-pt
+  "TRY", #https://docs.astral.sh/ruff/rules/#tryceratops-try
+  "FLY", #https://docs.astral.sh/ruff/rules/#flynt-fly
+  "PL", #https://docs.astral.sh/ruff/rules/#pylint-pl
+  "NPY", #https://docs.astral.sh/ruff/rules/#numpy-specific-rules-npy
+  "RUF", #https://docs.astral.sh/ruff/rules/#ruff-specific-rules-ruf
+]
+ignore = ["S101", "E731", "E501"]
```

Comment:
Do we have any `assert` for the runtime code?

Reply:
Not in src but in tests

Reply:
It should be allowed to use `assert` in tests. Seems it's a bug in ruff https://github.com/astral-sh/ruff/issues/4368.